p4-16/spec/P4-16-spec.mdk

```diff
@@ -7556,6 +7571,7 @@ The following table shows all P4 reserved annotations.
 | `optional`    | parameter is optional                              | [#sec-optional-parameter-annotations] |
 | `tableonly`   | action cannot be a default_action                   | [#sec-table-action-anno] |
 | `deprecated`  | Construct has been deprecated                      | [#sec-deprecated-anno]   |
+| `noWarn`      | Has a string argument; inhibits compiler warnings  |                          |
```

Comment:
Seems it might be nice to include cross-reference to the section, as is currently done for other annotations.